File: src/com/github/quarck/smartnotify/ToggleMuteBroadcastReceiver.java
package com.github.quarck.smartnotify;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class ToggleMuteBroadcastReceiver extends BroadcastReceiver
{
	@Override
	public void onReceive(Context context, Intent intent)
	{
		GlobalState.setIsMuted(context, ! GlobalState.getIsMuted(context));
		
		OngoingNotificationManager.updateNotification(context);
	}
}

File: app/src/main/java/com/github/quarck/smartnotify/ToggleMuteBroadcastReceiver.kt
package com.github.quarck.smartnotify

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class ToggleMuteBroadcastReceiver : BroadcastReceiver()
{
	override fun onReceive(context: Context, intent: Intent)
	{
		GlobalState.setIsMuted(context, !GlobalState.getIsMuted(context))

		OngoingNotificationManager.updateNotification(context)
	}
}
